Documentation/Templating/Index.rst

```diff
@@ -0,0 +1,72 @@
+.. include:: ../Includes.txt
+
+
+.. _Templating:
+
+==========
+Templating
+==========
+
+At this point, you should now be able to install and configure TYPO3 and be familiar with
+its backend. You may be wondering how you go about adding bespoke HTML templates
+to your own TYPO3 project and you might have noticed that the HTML, CSS and Javascript for the Introduction Package are
+are not accessible in the backend.
+
+Whilst this tutorial does not cover templating in TYPO3, this section is intended to answer both of these questions
+and give you a brief introduction to TYPO3's templating engine and also discuss best practices for managing your sites
+HTML templates, stylesheets and other assets.
+
+.. _Introducing Fluid:
+
+
+Introducing Fluid
+=================
+
+Fluid is TYPO3's templating engine. Fluid acts as the glue between your static HTML templates and the content you create in TYPO3's backend.
+
+A typical Fluid template will contain HTML that defines the structure of your page and Fluid tags that call various types of dynamic content.
+
+For example a simple web page that features a navigation menu, a few paragraphs of text and a company logo would contain three Fluid tags.
+
+- One to insert the content element that contains our text.
+- Another to call the Typoscript menu for our main navigation bar
+- And a third to insert the company logo
+
+Fluid's syntax is easy to learn and is very similar to HTML by design.
+
+
+.. _Where are templates stored:
+
+Where are templates stored
+==========================
+
+Fileadmin
```

Comment:
`Fileadmin` and `Sitepackages` should be sub sections of `Where are templates stored`? In that case they need a different sign to mark them as headlines, e.g. `-`